griptape/tools/google_cal/tool.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import datetime
+from schema import Schema, Literal, Optional
+from attr import define
+from griptape.artifacts import TextArtifact, ErrorArtifact, InfoArtifact
+from griptape.core.decorators import activity
+from griptape.tools import BaseGoogleClient
+
+
+@define
+class GoogleCalClient(BaseGoogleClient):
```

Comment:
Let's call it `GoogleCalendarClient` since that's the default name that will go to the LLM. "Cal" is slightly more confusing.